.gitignore

```diff
@@ -0,0 +1,3 @@
+/venv
```

Comment:
May need to add this as `venv/`

Also add `*.pyc` to this file

Reply:
done and pushed